Extractor/UAClient.cs

```diff
@@ -117,6 +123,12 @@ public async Task Run(CancellationToken token, int timeout = -1)
             await StartSession(timeout);
             await StartNodeMetrics();
         }
+
+        public async Task StartCustomServerSubscriptions(UAExtractor extractor)
```

Comment:
Does this need to be public at all? When are the node metrics subscriptions created?

Reply:
I'm not a huge fan of that logic. I think we should try to avoid this kind of circular references as much as possible, where

 - UAClient owns a class
 - That class owns a reference to UAExtractor
 - UAExtractor owns UAClient

Is there a reason why UAClient should own the subscription handler and not UAExtractor? Since this affects extractor behavior I think it belongs on the extractor more than the client.